packages/teleport/src/Main/Main.tsx

```diff
@@ -90,6 +90,14 @@ export function Main(props: State) {
     ctx.storeNav.getSideItems()[0]?.getLink(cfg.proxyCluster) ||
     cfg.routes.support;
 
+  const banners = props.alerts
```

Comment:
[nit]
You can destructure alerts from props and assign default [] there
```ts
const { status, statusText, ctx, alerts = [] } = props;
...
const banners = alerts.map(alert => ({
    message: alert.spec.message,
    severity: alert.spec.severity,
    id: alert.metadata.name,
}))
```

Reply:
Good catch, not sure why I did it like this 😆